src/styles/colors.less

```diff
@@ -1,137 +1,136 @@
-.primary-color {
+.primary-bg {
```

Comment:
Ah sorry, I meant this change for Quantum since we were creating new classes there. These might be used by other applications, so let's just stick with the current suffix.